sys/include/net/netopt.h

```diff
@@ -269,6 +269,18 @@ typedef enum {
      */
     NETOPT_LAST_ED_LEVEL,
 
+    /**
+     * @brief   Type for sequiential processing of IPv6 extension headers
+     *
+     * On GET it requests a thread to insert an extension header.
```

Comment:
Why does get not just return a `gnrc_pktsnip_t` and the IPv6 thread inserts the header?